### ISSUE TITLE
feature: Open external links clicked in Chromeless app in default browser

### ISSUE THREAD

**Issue by jovobe** [webcatalog/chromeless#54]
# Feature Request
**Specifications:**
- App Version: v2.0.0-beta
- OS (macOS, Windows, Linux): macOS 11.5.1

**Description:**
In the following example I have created a Gmail app with Chromeless. When I click on a link (lets say https://news.ycombinator.com/) inside an email I would like the Gmail app to open that link in my default browser, not in the Gmail app created with Chromeless.

**Additional Context**
This feature is a key for me for using apps created with Chromeless. I am currently using Epichrome, and it solves this issue with a Chrome helper extension which is installed inside the created (in this example Gmail) app.

**Comment by grouchal** [comment]
I agree this reduces the use of chromeless app if I can't open a URL from an email in that app.
